index.js

```diff
@@ -44,8 +44,18 @@ function uninstallDebugContextMenu(win) {
 	win.webContents.removeListener('devtools-closed', deactivateDebugContextMenu);
 }
 
-module.exports = () => {
+module.exports = opts => {
+	opts = opts || {
+		toggle: false
+	};
```

Comment:
There's no need to have default value for `false` as it's undefined by default which coerces to false.